.github/workflows/csv-validate.yml

```
@@ -1,13 +1,12 @@
-name: Validate bundle CSV
+name: Validate Bundle CSV
 
 on:
   pull_request:
     branches:
     - main
-    - 'release-*.*.*'
+    - 'release-*.*'
     paths:
-    - 'bundle/**'
-    workflow_dispatch:
+    - 'config/**'
```

```suggestion
    - 'config/**'
    - 'bundle/**'
```
might make sense to check both.. since one might accidentally only update `bundle/` and not `config`